src/components/Filters/FiltersWithRedux.jsx

```diff
@@ -0,0 +1,214 @@
+import "./Filters.css";
+import { Gridd } from "../Grid/Grid";
+import { useEffect } from "react";
+import { useDispatch, getState } from "react-redux";
+
+
+import {
+  filterByCategory,
+  filterByRating,
+  getAllData,
+  filterLowtoHigh,
+  filterHightoLow,
+} from "../../redux/Slice";
+
+
+const Axios = require("axios");
```

Comment:
```suggestion
const axios = require("axios");
```
we use camelCase for variables